src/main/js/field/TableDisplay.js

```diff
@@ -50,24 +53,35 @@ class TableDisplay extends Component {
         this.updateData();
     }
 
-    componentDidUpdate(prevProps, prevState) {
-        if (!this.state.showConfiguration && this.state.currentRowSelected && !this.props.inProgress && !this.props.hasFieldErrors && this.state.uiValidation === VALIDATION_STATE.SUCCESS) {
+    componentDidUpdate() {
+        const { showConfiguration, currentRowSelected, uiValidation } = this.state;
+        const { inProgress, hasFieldErrors } = this.props;
+        if (!showConfiguration && currentRowSelected && !inProgress && !hasFieldErrors
+            && uiValidation === VALIDATION_STATE.SUCCESS) {
             this.handleClose();
         }
     }
 
     createTableColumns() {
         const defaultDataFormat = (cell) => {
             if (cell) {
-                return <div title={cell.toString()}> {cell} </div>;
+                return (
+                    <div title={cell.toString()}>
+                        {cell}
+                    </div>
+                );
             }
-            return <div> {cell} </div>;
+            return (
+                <div>
+                    {cell}
+                </div>
+            );
         };
 
-        return this.props.columns.map(column => {
-
-            const assignedDataFormate = column['dataFormat'] ? column['dataFormat'] : defaultDataFormat;
-            const searchable = column['searchable'] ? column['searchable'] : true;
+        const { columns } = this.props;
+        return columns.map((column) => {
+            const assignedDataFormate = column.dataFormat ? column.dataFormat : defaultDataFormat;
+            const searchable = column.searchable ? column.searchable : true;
```

Comment:
I know this was already here, but won't `searchable` always be `true` because of this? Similar question with `dataFormat` above.

Reply:
The dataFormat attribute is meant to be a function to be used for rendering.  So the if check determines if the property is set otherwise use the default format.  Since searchable is meant to be a boolean that is a problem.  I need to change the logic to see if the property is defined otherwise default to true.